packages/devtools_app/lib/src/screens/performance/tabbed_performance_view.dart

```diff
@@ -119,22 +132,6 @@ class _TabbedPerformanceViewState extends State<TabbedPerformanceView>
     final hasData = data != null && !data.isEmpty;
     final searchFieldEnabled = hasData && !widget.processing;
     return [
-      _buildTab(
-        tabName: 'Timeline Events',
```

Comment:
moving this tab to no longer be the default seems like a change we should consult UXR on @InMatrix. I would thing at this point, our users expect to see the timeline trace when they open the perf page. I guess the question we need to ask is, what do we want our users to be looking at by default?

Reply:
My general thought is that tabs offering higher level information should be shown first. One question that can potentially help us figure out how prominent the Timeline Events tab still needs to be is: _what kinds of performance problems users still have to use the Timeline Events tab, instead of any of the other three tabs, to understand?_

Reply:
I think users should only go to the timeline tab after they have verified their problem cannot be solved by another tab.
For example, if you have a raster issue, use the frame analysis tab before even thinking about looking at the timeline tab. If you have a build problem, make sure you aren't just building things you didn't expect before drilling into the timeline tab.